src/components/Todos/TodoList.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  preparedTodos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ preparedTodos }) => (
+  <ul className="todos">
+    {preparedTodos.map(element => (
+      <li key={element.id} className="todos__list">
```

Comment:
todos => todo
users => user
books => book

avoid `element` word
```suggestion
    {todos.map(todo => (
      <li key={element.id} className="todos__list">
```